### DISCUSSION TITLE
第1节课笔记(13班)

### DISCUSSION THREAD

**Discussion by Lum1104** [InternLM/Tutorial#55] [笔记]
第一节课笔记（13班）👇

**Comment by Tianhang-Lu** [comment]
https://zhuanlan.zhihu.com/p/675922550

**Reply by Lum1104** [reply]
9分

**Comment by youqi111** [comment]
https://www.yuque.com/g/youqi-pyncc/if61gr/ushxfh4bzl2ynuzz/collaborator/join?token=sp6XsVkm8wtwBuaQ&source=doc_collaborator# 《书生·浦语大模型全链路开源体系》

**Reply by Lum1104** [reply]
wow手写笔记！10分！

**Comment by Taixi-Feng** [comment]
https://zhuanlan.zhihu.com/p/675930030

**Reply by Lum1104** [reply]
10分！

**Comment by NiklausPoel** [comment]
13班老谢：https://mw81w5z3eyw.feishu.cn/wiki/CRPdwfMQTi03s4kIg4scJ05FnGe?from=from_copylink

**Reply by Lum1104** [reply]
不明白的名词可以去网上搜搜资料呀，课程只是带大家入门～

**Reply by Lum1104** [reply]
8分

**Comment by bianchengyuanyuan** [comment]
https://ou3v176g8k.feishu.cn/docx/K0PMdVdBSoF0zfxRCLDchST9nCd#第一节课笔记

**Reply by Lum1104** [reply]
8分，继续加油哈

**Comment by qingyin741** [comment]
https://zhuanlan.zhihu.com/p/676033945

**Reply by Lum1104** [reply]
9分

**Comment by suwief** [comment]
https://zhuanlan.zhihu.com/p/676080300

**Reply by Lum1104** [reply]
9分

**Comment by SiliconSisyphus** [comment]
https://colorful-liquid-604.notion.site/60f62dcdd0a548beacc03a02969d5ac2?pvs=4

**Reply by Lum1104** [reply]
10分

**Comment by utopia2077** [comment]
https://nibqzjipp3p.feishu.cn/docx/Z5vKddwyRoR3F6xEF9Ncx52znxe

**Reply by Lum1104** [reply]
10分～

**Comment by huiruije** [comment]
https://blog.csdn.net/m0_59356572/article/details/135405464?spm=1001.2014.3001.5502

**Reply by Lum1104** [reply]
9分～

**Comment by qinb** [comment]
https://docs.google.com/document/d/1RIKSrrGgb_9fI2uv-VffghW4ao0Cv3nbRtk1A_6etAY/edit?usp=sharing

**Comment by 5Fbw** [comment]
https://fifth-tower-f54.notion.site/1298d29dde7c47f4907fe182b5a88a1d?pvs=4

**Comment by BOOMO4** [comment]
https://l055jh6h54r.feishu.cn/wiki/HL47w7ZSLipP8zkRxyMcObRFn3f?from=from_copylink

**Comment by burningmysoul2077** [comment]
762 - https://github.com/burningmysoul2077/Notes/blob/main/Datawhale%E7%BB%84%E9%98%9F%E5%AD%A6%E4%B9%A0/%E4%B9%A6%E7%94%9F%E6%B5%A6%E8%AF%AD%E5%AE%9E%E6%88%98/%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%85%A8%E9%93%BE%E8%B7%AF%E5%BC%80%E6%BA%90%E5%BC%80%E6%94%BE.md

**Comment by Loewen-Hob** [comment]
http://t.csdnimg.cn/KwoEb
第一课笔记——InternLM大模型全链路开源体系

**Comment by LouisCanBe** [comment]
https://www.yuque.com/selfmotivation-wxl/cvk32b/opobwnkf3149f2zl?singleDoc# 《01书生·浦语大模型全链路开源体系》/13班Louis

**Comment by zss019** [comment]
[第一讲-书生·浦语大模型全链路开源体系.md](https://github.com/InternLM/tutorial/files/13841455/-.md
第一节课笔记——书生·浦语大模型全链路开源体系

**Reply by Lum1104** [reply]
`[第一讲-书生·浦语大模型全链路开源体系.md](https://github.com/InternLM/tutorial/files/13841455/-.md)`

**Comment by RessCris2** [comment]
第一课笔记 https://mp.csdn.net/mp_blog/creation/success/135416724

**Reply by RessCris2** [reply]
https://weifei.blog.csdn.net/article/details/135416724 更正链接

**Comment by qingtao12138** [comment]
第一课主要介绍了书生浦语的全链条大模型开放体系，包括数据、预训练、微调、评测、部署和智能体等六个模块，并对每个模块的功能和特点进行了详细的说明：

1. 大模型发展：大模型是通用人工智能的关键，能够同时处理多种任务和多种模态的数据，如文本、图像、视频等；
2. InternLM系列模型：这是一系列高性能的开源大模型，规模从7B到130B不等，其中20B模型在多个能力评测上都超过了同类模型，甚至在C-Eval和阅读理解等任务上还击败了Llama-70B模型；
3. 大模型应用路径图：这是一个指导大模型在生产环境中应用的框架，需要根据不同的模型特点、业务场景、环境交互等因素进行综合考量和优化；
4. 书生万卷：这是一个包含多模态高质量数据的开放平台，数据总量达到2T，涵盖了文本、图像、视频等多种形式，平台名称为OpenDataLab；
5. InternLM-Train：这是一个用于预训练大模型的工具，具有高可扩展性和兼容性，支持并行训练，相比deepspeed能够提升50%的训练速度；
6. XTuner：这是一个用于高效微调大模型的工具，采用了如Lora等先进的算法，能够降低显存消耗，例如7B模型只需8GB显存就可以微调，同时支持多种微调任务和算法；
7. OpenCompass：这是一个用于评测大模型能力的开放平台，提供了6个维度、80多个评测集、40万多个评测题目，覆盖了学科、语言、推理、安全等多个领域，并构建了一个开源的评测系统；
8. LMDeploy：这是一个基于FT的高效部署工具，部署效率领先vllm，提供了并行、量化、Attention优化、KVcache优化和continuous batching等功能，还能够将推理引擎接入opencompass评测体系；
9. Lagent：这是一个轻量级的智能体框架，支持多种类型的智能体能力，如ReAct和AutoGPT等，能够缓解大模型知识的局限性；
10. AgentLegp：这是一个多模态智能体工具箱，定义了一套可扩展的工具接口，同时可以适配多个主流的智能体系统；

**Comment by QingyanMu** [comment]
【InternLM大模型全链路开源体系 - CSDN App】http://t.csdnimg.cn/GaoYE

**Comment by Xrcb** [comment]
http://t.csdnimg.cn/60XYg
